ops/infrastructure/modules/efs-filesystem/output.tf

```diff
@@ -0,0 +1,55 @@
+################################################################################
+# File System
+################################################################################
+
+output "arn" {
+  description = "Amazon Resource Name of the file system"
+  value       = module.efs.arn
+}
+
+output "id" {
+  description = "The ID that identifies the file system (e.g., `fs-ccfc0d65`)"
+  value       = module.efs.id
+}
+
+output "dns_name" {
+  description = "The DNS name for the filesystem per [documented convention](http://docs.aws.amazon.com/efs/latest/ug/mounting-fs-mount-cmd-dns-name.html)"
+  value       = module.efs.dns_name
+}
+
+output "size_in_bytes" {
+  description = "The latest known metered size (in bytes) of data stored in the file system, the value is not the exact size that the file system was at any point in time"
+  value       = module.efs.size_in_bytes
+}
+
+################################################################################
+# Mount Target(s)
+################################################################################
+
+output "mount_targets" {
+  description = "Map of mount targets created and their attributes"
+  value       = module.efs.mount_targets
+}
+
+################################################################################
+# Security Group
+################################################################################
+
+output "security_group_arn" {
+  description = "ARN of the security group"
+  value       = module.efs.security_group_arn
+}
+
+output "security_group_id" {
+  description = "ID of the security group"
+  value       = module.efs.security_group_id
+}
+
+################################################################################
+# Access Point(s)
+################################################################################
+
+output "access_points" {
+  description = "Map of access points created and their attributes"
+  value       = module.efs.access_points
+}
```

Comment:
**issue(non-blocking):** This will output 2 long arrays in the command line containing surplus information.
**suggestion(non-blocking):**  How about just output the arn of the access point as below:
```
output "access_points_arns" {
  description = "Map of access points created and their attributes"
  value       = values(module.efs.access_points)[*].arn
}
```
Then update the `terraform.tf` as:
```
output "efs_filesystem_access_points_arns" {
  value = module.gigadb_efs.access_points_arns
}
```
Then the output would be:
```
efs_filesystem_access_points_arns = [
  "arn:aws:elasticfilesystem:ap-northeast-2:$owner_id:access-point/fsap-055e12f2852b55e46",
  "arn:aws:elasticfilesystem:ap-northeast-2::$owner_id:access-point/fsap-0b9493183af650e29",
]
```

Reply:
Hi @kencho51 

This is a very good idea, but the location and the suggested information are not quite right:

1) The module `efs-filesystem` might be re-used in other contexts where the required output differs, so we WANT the module to output everything it knows about access point.  The output segregation should be done in whatever uses the module, (`terraform.tf`) in our case.

2) To mount an EFS fileystem in an EC2 instance, it's not the ARN that's needed, but the ID.


In any case, I've made a commit  ([890e8ae](https://github.com/gigascience/gigadb-website/pull/1763/commits/890e8ae19cb897245637f7b540dc76a28ef48e9a)) to implement that.